Staging_Dev/harvestlog.sql

```diff
@@ -0,0 +1,43 @@
+USE [perpetuumsa]
+GO
+
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+
+CREATE TABLE [dbo].[harvestlog](

```

Comment:
To make this stable to re-run you can do:
```sql
DROP TABLE IF EXISTS [dbo].[harvestlog]
```
Not a strict requirement, but the more stable patches are the easier it is to dev against (in case changes are made and something needs to be re-run)

Reply:
I understand and did the modification